### ISSUE TITLE
Disable form submit if form is not valid

### ISSUE THREAD

**Issue by BacLuc** [ecamp/ecamp3#2896]
Currently, an invalid form can be submitted on:
- [x] Register.vue (fixed in #3028 and #3407)
- [x] Collaborators.vue (fixed in #3407)

**Comment by manuelmeister** [comment]
Accessibility issue: Form buttons should not be disabled, except when a blocking update action is dispatched. If the button is disabled, then it is not visible to screenreaders. Also it might not be self explanatory why the submit button is disabled or that it is disabled and not just not working.

Check the above cases, if the blocking action applies, otherwise make the button aria-disabled and add a tooltip to explain what needs to be done to make the form submittable.

**Comment by DeNic0la** [comment]
Register didn't have a disabled attribute on the Button. Collaborators did but it was reworked in #3348 and now blocks submit over the handle `handleSubmit` on `ValidationObserver` (like Register).
Is the aria-disabled buttons still something to pursue?  #3502 
what kind of text should be in the tooltip? A generic "Invalid data"-message or a summary of all the error messages from the Input fields? Is a tooltip even that helpful? If the submit button is pressed (even with aria-disabled (at least on my machine)) the invalid form fields get marked with error messages even if untouched. (Tested on "Invite people"-Dialog)
